### PR TITLE
Fix redemption

### PR DESCRIPTION
**Pull request by igorline**

### CODE REVIEW

**Comment by milkyklim** [comment]
Need to replace these two images that are shown after redeem flow:
- [one](https://github.com/unisocksclassic/unisocksclassic/blob/redemption-classic/src/components/Gallery/sent.png)
- [two](https://github.com/unisocksclassic/unisocksclassic/blob/redemption-classic/src/componentsGallery/nfc.png)

**Comment by milkyklim** [comment]
Should be final and done.

**Comment by igorline** [comment]
@milkyklim, ~~done!~~

**Comment by igorline** [comment]
We need to fix #11 #12 to get this done

**Comment by milkyklim** [comment]
This commit should cover #11.

File: src/utils/index.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ export const TOKEN_ADDRESSES = {
 // Rinkeby
 export const REDEEM_ADDRESS = '0x3c2f192ee7faad7a8818d2611ce4c9a676b77e7f'
 
+export const IS_MAINNET = (REDEEM_ADDRESS == '0x3c2f192ee7faad7a8818d2611ce4c9a676b77e7f' ? false : true)
```
Comment (igorline) [inline src/utils/index.js]: This you'd rather get from web3 provider

Comment (milkyklim) [reply src/utils/index.js]: Fix incoming!

**Comment by igorline** [comment]
@milkyklim will you merge this?

**Comment by milkyklim** [comment]
@igorline Have a look. I am ready to merge.

**Comment by igorline** [comment]
LGTM

**Pull request merged**